src/components/SpecificTopicArticles.jsx

```diff
@@ -1,4 +1,6 @@
-import { useParams } from "react-router-dom";
+// MAKE TOPIC A QUERY ON A REFACTOR SO THAT EVERYTHING IS CONTROLLED BY THE HOME PAGE - TOO MUCH COPY AND PASTE
```

Comment:
I was going to suggest you do this, so I'm glad you've got a note in here to do it later! 😄